packages/frontend/core/src/components/affine/setting-modal/general-setting/billing/index.tsx

```diff
@@ -108,8 +108,8 @@ const SubscriptionSettings = () => {
       ? '0'
       : price
         ? recurring === SubscriptionRecurring.Monthly
-          ? String(price.amount / 100)
-          : String(price.yearlyAmount / 100)
+          ? String((price.amount ?? 0) / 100)
+          : String((price.yearlyAmount ?? 0) / 100)
```

Comment:
should we hide non-released plan at frontend?

Reply:
backend won't send non-released plans